src/commands/txCommands/util/index.js

```diff
@@ -208,7 +210,13 @@ const uploadTrnsMaster$ = ([lang_tag, content]) =>
 		MASTER_RESOURCE,
 		lang_tag,
 		resourceContent(MASTER_RESOURCE, content)
-	);
+	).map(response => [lang_tag,response])
```

Comment:
I don't think you need this `.map` since `lang_tag` is already in-scope in the `do` as well.